### DISCUSSION TITLE
How to write custom metadata (properties)

### DISCUSSION THREAD

**Discussion by PetrKaleta** [gettalong/hexapdf#238] [Q&A] [answered]
Hello,
is there any way to write custom metadata (properties) like `document-id` to a PDF file? Like in Acrobat 
![prop](https://user-images.githubusercontent.com/282588/225711429-a7edf6fe-9a94-426d-ae17-19b4096494bb.jpeg)

**Comment by gettalong** [comment, marked as answer]
Yes, you can do this. Provide a file with the metadata in it and I can tell you how with HexaPDF. Usually it boils down to something like `document.trailer.info[:'document-id'] = 'd4....27e'` but it may be different depending on the application you are using to create the meta data.

**Reply by PetrKaleta** [reply]
Thanks that was exactly what I was looking for
